src/@ractf/api/challenge.js

```diff
@@ -36,6 +36,12 @@ export const resyncChallenge = (id) => {
     });
 };
 
+export const updateChallengeLocal = (data) => {
+    // data in the form { challenge_id: number, score: number, name: string } etc
+    store.dispatch(actions.editChallenge(data));
+    return data;
```

Comment:
Why does this return `data` if the only usage doesn't use the return value? It is highly likely I don't understand JavaScript.

Reply:
good question, the rest of them do and this codebase doesn't use TypeScript so I'm too scared not to!